### PR TITLE
Update CRSP to use gRPC and Pain.001

### PR DESCRIPTION
**Pull request by johanfol**

### CODE REVIEW

File: channel-router-setup-processor/src/index.ts [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+  const appService: ApplicationService = new ApplicationService();
+
+  /**
+ * KOA Rest Server
+ */
+  const app = new Koa();
+
+  const router = new Router();
+
+  router.get(/.*/gim, async (ctx) => {
```
Comment (joeygoksu) [inline channel-router-setup-processor/src/index.ts:31]: Can you separate the file like in the tms service?
Router, Server, App etc..

File: channel-router-setup-processor/src/index.ts [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+    // app.listen(config.restPort, () => {
+    //   console.log(`API restServer listening on PORT ${config.restPort}`);
+    // });
+
+    console.log(`gRPC:Server:${bindPort}`, new Date().toLocaleString());
+    server.start();
+  });
+
+}
+try{
```
Comment (joeygoksu) [inline channel-router-setup-processor/src/index.ts:73]: Can you format this line with Prettier

**Review by joeygoksu — CHANGES_REQUESTED** [review]

File: channel-router-setup-processor/package.json [hunk 3]
```diff
@@ -3,42 +3,59 @@
   "version": "0.1.0",
```
Comment (joeygoksu) [inline channel-router-setup-processor/package.json:3]: Can you update the lint-staged end of lines? There are multiple linting and formating errors on the repo. 

`"lint-staged": {
    "*.{js,ts}": [
      "eslint --fix",
      "prettier --write"
    ]
  }`

**Review by joeygoksu — CHANGES_REQUESTED** [review]

File: channel-router-setup-processor/package.json (hunk 3 above)
Comment (johanfol) [reply channel-router-setup-processor/package.json:3]: This throws error - but I did run linting and fixed all the issues.

**Review by joeygoksu — APPROVED** [review]

**Comment by joeygoksu** [comment]
Well done, mate 🥳

**Pull request merged**